src/modules/authentication/components/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,7 +1,60 @@
+import { useForm } from "react-hook-form";
+import authLogo from '../../../../assets/images/logo-pms.png'
+import { toast } from "react-toastify";
+import { useNavigate } from "react-router-dom";
+import { AUTH_URL, axiosInstanceURL } from "../../../../services/EndPoints";
+
 export default function ForgetPassword() {
+  const Navigate = useNavigate()
+  const {
+    register,
+    formState: { errors, isSubmitting },
+    handleSubmit,
+  } = useForm();
+
+  const onSubmit = async (data: any) => {
+    try {
+      const res = await axiosInstanceURL.post(AUTH_URL.FORGET_PASSWORD, data)
+      console.log(res);
+      Navigate('/reset-password', {state: {email: data.email}})
+    } catch (error: any) {
+      toast.error(error.response.data.message)      
+    }    
+  }
+
   return (
-    <div>
-      <h1>Forget Password</h1>
+    <div className='col-lg-4 col-md-8 col-sm-12'>
```

Comment:
We can make the AuthLayout having repeated structure and take the image dynamically via props or using useLocation hook to show the correct image.